src/SDKs/DevTestLabs/DevTestLabs.Tests/Tests/BasicTests.cs

```diff
@@ -20,7 +20,7 @@ public void ListLabsTest()
             {
                 var client = GetDevTestLabsClient(context);
 
-                var labs = client.Lab.ListBySubscription().ToList();
+                var labs = client.Labs.ListBySubscription().ToList();
```

Comment:
@ryanvog are you asking for exception for adding test coverage? If yes, that will not be possible, unless you have a good reason not to add any test coverage.